widgyts/dataset_viewer.py

```diff
@@ -199,9 +200,30 @@ def widget(self):
 
         traitlets.link((dropdown, "value"), (self, "grid_colormap"))
 
+        button = ipywidgets.Button(description="Positions")
```

Comment:
The button should have some more descriptive description, perhaps something like "Add Keyframe"?